src/components/creation/CharacterCard.tsx

```diff
@@ -1,33 +1,31 @@
-import { Link, useNavigate } from "react-router-dom"
+import { useNavigate } from "react-router-dom"
 import { AbilityScore } from "../../model/Character/AbilityScore";
 import { CreatedChar } from "../../model/Character/CreatedChar.interface"
 import { deleteCharacter } from "../controller";
 import '../styles/characterCard.css'
-
-
 interface IProps {
     character: CreatedChar
 }
 
-
 function CharacterCard({character}: IProps) {
-
+    const scoreTitles = ['Str', 'Dex', 'Con', 'Intl', 'Wis', 'Cha'];
```

Comment:
What do you think about putting these in it's own ts file and exporting it so that the whole app can use them?